### PR TITLE
Databricks plugin

### PR DESCRIPTION
**Pull request by pingsutw**
Signed-off-by: Kevin Su <pingsutw@apache.org>

## Describe your changes
- Add databricks config to the charts
- Add databricks config to plugin setup page

### CODE REVIEW

File: charts/flyte-core/values.yaml [hunk 1]
```diff
@@ -821,3 +821,18 @@ sparkoperator:
...
+
+# -- Optional: Databricks Plugin allows us to run the spark job on the Databricks platform.
+databricks:
+  enabled: false
+  plugin_config:
+    plugins:
+      databricks:
+        entrypointFile: dbfs:///FileStore/tables/entrypoint.py
+        # Databricks account
+        databricksInstance: dbc-a53b7a3c-614c
```
Comment (tmathur-hbo) [inline charts/flyte-core/values.yaml:838]: How do we deal with multiple databricks environments? I guess you are assuming one environment of dbx. Is there an ability to overwrite this value at workflow submission?

File: deployment/sandbox/flyte_helm_generated.yaml [hunk 2]
```diff
@@ -7962,3 +7862,4 @@ spec:
   parallelism: 1
   completions: 1
   backoffLimit: 1
+>>>>>>> 8ac8aae20eb7081a0455dd883b00134af3fabee0
```
Comment (tmathur-hbo) [inline deployment/sandbox/flyte_helm_generated.yaml:7865]: remove?

File: charts/flyte-core/values.yaml (hunk 1 above)
Comment (pingsutw) [reply charts/flyte-core/values.yaml]: I though you only have one dbx account. Let me add it to the dbx config on flytekit-side

File: deployment/sandbox/flyte_helm_generated.yaml (hunk 2 above)
Comment (pingsutw) [reply deployment/sandbox/flyte_helm_generated.yaml]: removed it, thanks

File: charts/flyte-core/values.yaml [hunk 3]
```diff
@@ -821,3 +821,18 @@ sparkoperator:
...
+# Optional Plugins
+# --------------------------------------------------------
+
+# -- Optional: Databricks Plugin allows us to run the spark job on the Databricks platform.
+databricks:
+  enabled: false
+  plugin_config:
+    plugins:
+      databricks:
+        entrypointFile: dbfs:///FileStore/tables/entrypoint.py
```
Comment (wild-endeavor) [inline charts/flyte-core/values.yaml:836]: is there an example of this file?

**Review by wild-endeavor — APPROVED** [review]

**Review by wild-endeavor — DISMISSED** [review]

**Review by wild-endeavor — APPROVED** [review]

**Pull request merged**